examples/src/pages/Viewer.tsx

```diff
@@ -89,6 +89,7 @@ export function Viewer() {
         domElement: canvasWrapperRef.current!,
         onLoading: progress,
         logMetrics: false,
+        useFlexibleCameraManager: true,
```

Comment:
Can we remove this from the PR? It will be a little obstructive to others that are testing in the examples

Reply:
Yes